src/Interfaces/Options/Shape/IPolygonShape.ts

```diff
@@ -0,0 +1,8 @@
+export interface IPolygonShape {
+    /**
+     * @deprecated use the new sides instead
+     */
+    nb_sides: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'nb_sides' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)